### PR TITLE
fix: add tests for /news in slug

### PR DESCRIPTION
**Pull request by raisedadead**

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -1,31 +1,36 @@
-const assert = require('assert').strict;
-const fs = require('fs');
-const redirects = JSON.parse(fs.readFileSync('./redirects.json', 'utf8'));
+const assert = require("assert").strict;
+const fs = require("fs");
+const redirects = JSON.parse(fs.readFileSync("./redirects.json", "utf8"));
 const fromSlugs = [];
+const newsSubPathRx = /^\/news/;
```
Comment (scissorsneedfoodtoo) [inline test.js:5]: const newsSubPathRx = /^\/news\//;

Comment (raisedadead) [reply test.js:5]: Nice catch.

**Comment by raisedadead** [comment]
Updated with your recommendations.

**Comment by scissorsneedfoodtoo** [comment]
Awesome, thanks for creating the test and making this update.

**Pull request merged**